src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -62,7 +62,27 @@ internal static void ShowUpdateNotification(PSHostUserInterface hostUI)
                     ? ManagedEntranceStrings.StableUpdateNotificationMessage
                     : ManagedEntranceStrings.PreviewUpdateNotificationMessage;
 
-                string notificationMsg = string.Format(CultureInfo.CurrentCulture, notificationMsgTemplate, releaseTag);
+                string notificationColor = string.Empty;
+                string resetColor = string.Empty;
+
+                string line2Padding = string.Empty;
+                string line3Padding = string.Empty;
+
+                // We calculate how much whitespace we need to make it look nice
+                if (hostUI.SupportsVirtualTerminal)
+                {
+                    // Use inverse yellow
+                    notificationColor = "\x1B[7m\x1B[1;33m";
```

Comment:
We got a lot of feedback in the [Emphasis PR](https://github.com/PowerShell/PowerShell/pull/8963) that using color that couldn't be configured is not accessible so we shouldnt do that here.

We can invert the colors of foreground and background, but the yellow is probably not a good way to proceed.

Reply:
It's a shame the blink escape sequence isn't as wildly supported 😉

(I'm kidding)

Reply:
Agreed (on the color; who uses blink? 😋)

Until we get some kind of configurability built into the engine in terms of which colors are used for which purposes, we need to continue to be mindful that we could be making it _harder_ to read for folks who may have some form of color blindness and verify the color is absolutely appropriate for the purpose.

Reply:
The problem is that the console host hasn't actually started yet, so I can't use $host.PrivateData to get a color like Warning color.  I suppose I can just invert the text and not use color if (like someone has a bright background).